### PR TITLE
fix(config): add missing parameters to Zooz ZEN30

### PR DESCRIPTION
**Pull request by shbatm**
Update Zooz ZEN30 parameter options to include paramaters 28, 29, and 30, added in recent firmware updates.

https://www.support.getzooz.com/kb/article/468-zen30-double-switch-advanced-settings/
https://www.support.getzooz.com/kb/article/393-zen30-double-switch-change-log/

### CODE REVIEW

File: packages/config/config/devices/0x027a/zen30.json [hunk 1]
```diff
@@ -470,6 +470,49 @@
 					"value": 2
 				}
 			]
+		},
+		{
+			"#": "28",
+			"$if": "firmwareVersion >= 1.2",
+			"label": "Enable/Disable Scene Control for Dimmer",
+			"description": "Enable or disable scene control functionality for quick double tap triggers on the dimmer.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen30.json]:
```suggestion
			"label": "Scene Control (Dimmer)",
```

File: packages/config/config/devices/0x027a/zen30.json [hunk 2]
```diff
@@ -470,6 +470,49 @@
...
+					"label": "Scene control enabled",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "29",
+			"$if": "firmwareVersion >= 1.2",
+			"label": "Enable/Disable Scene Control for Relay",
+			"description": "Enable or disable scene control functionality for quick double tap triggers on the relay button.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen30.json]:
```suggestion
			"label": "Scene Control (Relay)",
```

File: packages/config/config/devices/0x027a/zen30.json [hunk 3]
```diff
@@ -470,6 +470,49 @@
...
+			"valueSize": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Scene control disabled",
+					"value": 0
+				},
+				{
+					"label": "Scene control enabled",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen30.json]:
```suggestion
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
```

File: packages/config/config/devices/0x027a/zen30.json [hunk 4]
```diff
@@ -470,6 +470,49 @@
...
```
Comment (AlCalzone) [inline packages/config/config/devices/0x027a/zen30.json]:
```suggestion
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
A few changes to comply with the style guide please. I'm not sure how obvious it is that the scenes can be triggered with double taps. If that's a concern, we can mention it in the description, but we should avoid writing "enable/disable" tree times.

**Comment by shbatm** [comment]
> A few changes to comply with the style guide please. I'm not sure how obvious it is that the scenes can be triggered with double taps. If that's a concern, we can mention it in the description, but we should avoid writing "enable/disable" tree times.

Sounds good, I just copied from the ZOOZ release notes.

**Review by AlCalzone — APPROVED** [review]

**Comment by kernelpanic85** [comment]
I have some ZEN 30 switches running the new firmware and took a look after deploying the update. The new options show under Configuration undefined rather than with the other options under Configurationv1. Also they aren't showing any names or descriptions. Screenshot below:

image

**Comment by AlCalzone** [comment]
Did you re-interview your device?

**Comment by kernelpanic85** [comment]
Ah, that did it, thanks!

image

**Pull request merged**